uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -46,14 +51,14 @@
  * @author Shawn Connolly, sconnolly@unicon.net
  */
 @Entity
-@Table(name = "UP_PAGS_TEST")
+@Table(name = JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX)
```

Comment:
Yes, it's a little weird.

But the table is pre-existing, and I imagine it's safer to keep the name as-is.  I could make the prefix simply `UP_PAGS_`, but then all of the table names would start with `TEST`.